zjit.h

```diff
@@ -14,23 +14,25 @@ extern bool rb_zjit_enabled_p;
 extern uint64_t rb_zjit_call_threshold;
 extern uint64_t rb_zjit_profile_threshold;
 void rb_zjit_compile_iseq(const rb_iseq_t *iseq, rb_execution_context_t *ec, bool jit_exception);
-void rb_zjit_profile_insn(enum ruby_vminsn_type insn, rb_execution_context_t *ec);
+void rb_zjit_profile_insn(int insn, rb_execution_context_t *ec);
```

Comment:
This seems to effectively hold for our supported platforms, but it still seems implementation-dependent in C in general. If you want to do this, can you update the signature of `rb_zjit_profile_insn` accordingly and explicitly cast it inside the Rust function?

> Each enumerated type shall be compatible with char, a signed integer type, or an unsigned integer type. The choice of type is implementation-defined, but shall be capable of representing the values of all the members of the enumeration.

Reply:
Updated 👍